creator-node/src/diskManager.ts

```diff
@@ -79,160 +87,6 @@ export async function getTmpTrackUploadArtifactsPath() {
-  if (!dirName || !fileName) {
-    genericLogger.error(
-      `Invalid dirName and/or fileName, dirName=${dirName}, fileName=${fileName}`
-    )
-    throw new Error('Must pass in valid dirName and fileName')
-  }
-
-  try {
-    CID.isCID(new CID(dirName))
-    CID.isCID(new CID(fileName))
-  } catch (e: any) {
-    genericLogger.error(
-      `CID invalid, dirName=${dirName}, fileName=${fileName}, error=${e.toString()}`
-    )
-    throw new Error(
-      `Please pass in a valid cid for dirName and fileName. Passed in dirName: ${dirName} fileName: ${fileName} ${e.message}`
-    )
-  }
-}
-
-/**
- * Given a directory name and a file name, construct the full file system path for a directory and a folder inside a directory.
- * This function does not ensure the path returned exists on disk, so use it for read-only operations
- * (like /ipfs retrieval) or tasks where a subsequent step will already ensure the path exists (like syncs).
- *
- * eg if you're manually computing the file path to an file `Qmabcxyz` inside a dir `Qmdir123`, use this function to get the
- * path with both the dir and the file /file_storage/files/r12/Qmdir123/Qmabcxyz
- * Use `computeFilePath` if you just want to get to the path of a file or directory.
- *
- * @param {String} dirName directory name
- * @param {String} fileName file name
- */
-export function computeFilePathInDir(dirName: string, fileName: string) {
-  _validateFileAndDir(dirName, fileName)
-
-  const parentDirPath = computeFilePath(dirName)
-  const absolutePath = path.join(parentDirPath, fileName)
-  genericLogger.info(`File path computed, absolutePath=${absolutePath}`)
-  return absolutePath
-}
-
-/**
- * Use for operations where it's necessary to ensure that the path exists before using it.
- * @see computeFilePathInDir - does the same thing but also performs the equivalent of 'mkdir -p'
- */
-export async function computeFilePathInDirAndEnsureItExists(
-  dirName: string,
-  fileName: string
-) {
-  _validateFileAndDir(dirName, fileName)
-
-  const parentDirPath = await computeFilePathAndEnsureItExists(dirName)
-  const absolutePath = path.join(parentDirPath, fileName)
-  genericLogger.info(`File path computed, absolutePath=${absolutePath}`)
-  return absolutePath
-}
-
-/**
- * Given a directory path, this function will create the dirPath if it doesn't exist
- * If it does exist, it will not overwrite, effectively a no-op
- * @param {*} dirPath fs directory path to create if it does not exist
- */
-export async function ensureDirPathExists(dirPath: string) {
-  try {
-    // the mkdir recursive option is equivalent to `mkdir -p` and should created nested folders several levels deep
-    await fs.mkdir(dirPath, { recursive: true })
-  } catch (e: any) {
-    genericLogger.error(
-      `Error making directory, dirName=${dirPath}, error=${e.toString()}`
-    )
-    throw new Error(`Error making directory at ${dirPath} - ${e.message}`)
-  }
-}
-
```

Comment:
I moved these all to fsUtils.ts (no changes otherwise) to fix integration test issues caused by order of imports (the order meant we couldn't stub a function)